sisl/viz/backends/matplotlib/backend.py

```diff
@@ -91,22 +94,24 @@ class MatplotlibMultiplePlotBackend(MatplotlibBackend, MultiplePlotBackend):
 
 class MatplotlibSubPlotsBackend(MatplotlibMultiplePlotBackend, SubPlotsBackend):
 
-    def draw(self, backend_info, rows, cols, childs, **make_subplots_kwargs):
+    def draw(self, backend_info):
+        childs = backend_info["child_plots"]
+        rows, cols = backend_info["rows"], backend_info["cols"]
 
-        self.figure, self.axes = plt.subplots(rows, cols)
+        self.figure, self.ax = plt.subplots(rows, cols)
```

Comment:
Why this name change? Axes is well describing multiple axis? (axis is singular?)

Reply:
Because it makes it easier to generalize methods, all matplotlib plots have the `self.ax` attribute and then an optional `ax_indices` can be passed. See:

https://github.com/pfebrer/sisl/blob/ccc1fcef193a4fe51ddda17533e3159174de106c/sisl/viz/backends/matplotlib/backend.py#L30-L53

I have not thought much about it though. I will think about it again tomorrow :)

Reply:
I think it would be best to use `axes` (plural) and `axis` (singular) and adhere to this naming scheme (everywhere).

Reply:
In `matplotlib` they always use `ax` (https://matplotlib.org/stable/tutorials/introductory/usage.html#sphx-glr-tutorials-introductory-usage-py), so I wanted to stick with `matplotlib` conventions for the `matplotlib` backend.

Reply:
Not internally. One thing is the user scripts, another thing is internally and how they adhere to objects.

Reply:
At least from the parts I read, see e.g. https://github.com/matplotlib/matplotlib/blob/9e0bb9c72121a7c3ba85c55368ae402bbfdd7782/lib/matplotlib/figure.py#L193.

Reply:
Hmm ok, but then it should always be `axes` (as they do internally from what I understand. Because in fact even if it's "singular" it contains the `x` axis and the `y` axis.

Reply:
True :)